fbgemm_gpu/include/fbgemm_gpu/hipcub_namespace_postfix.cuh

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
Can we merge this file with `cub_namespace_postfix.cuh`?

Reply:
These `hipcub_namespace_postfix.cuh`, `hipcub_namespace_postfix.hpp` are accidentally added in git. They should be removed. Thanks for reminding me.